.eslintrc.js

```diff
@@ -0,0 +1,81 @@
+module.exports = {
+  plugins: [
+    "@typescript-eslint",
+    "react",
+    "react-hooks",
+    "better-styled-components",
```

Comment:
I know it's a copy paste, but do we actually need this?

Reply:
deleted `react-hooks` as it's included in react-scripts:
https://www.npmjs.com/package/eslint-plugin-react-hooks

also deleted `better-styled-components` as we don't use styled-components